.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
You probably wanted to ignore this file 😄

Reply:
You're right 🤦 
Added `.vscode` to gitignore and removed the folder